src/extensions/default/HealthData/HealthDataManager.js

```diff
@@ -121,16 +120,41 @@ define(function (require, exports, module) {
 
                                 PreferencesManager.setViewState("UUID",      oneTimeHealthData.uuid);
                                 PreferencesManager.setViewState("OlderUUID", oneTimeHealthData.olderuuid);
-
                                 return result.resolve(oneTimeHealthData);
                             }
                         }
                     });
 
             });
-
         return result.promise();
     }
+    
+    // Get Analytics data
+    function getAnalyticsData() {
+        var userUuid = PreferencesManager.getViewState("UUID"),
+            olderUuid = PreferencesManager.getViewState("OlderUUID");
+
+        return {
+            project: brackets.config.serviceKey,
+            environment: "production",
```

Comment:
We might have to externalize this as when posting to stage this is going to be `environment: "stage"`